Pod/Classes/Terminals/Utility/GlobalMobileSDK/GMSRequestHelper.swift

```diff
@@ -52,7 +52,7 @@ class GMSRequestHelper {
         }
 
         if cardData != nil {
-            return AuthTransaction.auth(total: decimalToUint(total),
+            return AuthTransaction.auth(clientTransactionId: "", total: decimalToUint(total),
```

Comment:
The value for `clientTransactionId` should be obtained from the builder when present.